apps/condo/lang/en/en.json

```diff
@@ -1886,15 +1886,15 @@
   "notification.messages.TICKET_STATUS_OPENED.email.subject": "Hi, it's Doma! \uD83E\uDD96",
   "notification.messages.TICKET_STATUS_OPENED.push.title": "New ticket \uD83D\uDCA1",
   "notification.messages.TICKET_STATUS_IN_PROGRESS.email.subject": "We have news on ticket #{data.ticketNumber}! \uD83E\uDD96",
-  "notification.messages.TICKET_STATUS_IN_PROGRESS.push.title": "We have news on ticket #{data.ticketNumber}! \uD83E\uDD96",
+  "notification.messages.TICKET_STATUS_IN_PROGRESS.push.title": "Ticket news \uD83D\uDCA1",
```

Comment:
Why do you remove ticket number from title?

Reply:
this is according to the specifications from the product manager